docs/semgrep-assistant/getting-started.md

```diff
@@ -119,6 +119,32 @@ Semgrep Assistant messages only appear in your PR comments for rules that are se
   ![ Policies modes](/img/semgrep-assistant-comment.png)
 * You have selected PR/MR comments in **Semgrep AppSec Platform > Settings > Deployment** in the **Code** section.
 
+### Add Memories (beta)
+
+Assistant Memories allow AppSec teams and developers to tailor Assistant's remediation guidance to their organization's standards and defaults on a per-project, per-rule basis. Whenever Assistant gives a suggested fix,  users can provide feedback by adding custom instructions.
```

Comment:
is it `allow` or `allows` considering the feature is a single entity (i.e. "GitHub Teams is a tier..." and not "GitHub Teams are a tier")?